### PR TITLE
Add rss icon

### PR DESCRIPTION
**Pull request by mooreds**
Just add the RSS icon to the footer

### CODE REVIEW

File: site/_includes/_footer.liquid [hunk 1]
```diff
@@ -39,10 +39,11 @@
...
         <ul class="inline">
-          <li><a href="https://github.com/fusionauth" target="_blank"><i class="fab fa-github-square fa-3x"></i></a></li>
-          <li><a href="https://twitter.com/fusionauth" target="_blank"><i class="fab fa-twitter-square fa-3x"></i></a></li>
-          <li><a href="https://facebook.com/fusionauth" target="_blank"><i class="fab fa-facebook-square fa-3x"></i></a></li>
-          <li><a href="https://www.linkedin.com/company/fusionauth/" target="_blank"><i class="fab fa-linkedin fa-3x"></i></a></li>
+          <li><a href="https://github.com/fusionauth" target="_blank"><i class="fab fa-github-square fa-2x"></i></a></li>
+          <li><a href="https://twitter.com/fusionauth" target="_blank"><i class="fab fa-twitter-square fa-2x"></i></a></li>
+          <li><a href="https://facebook.com/fusionauth" target="_blank"><i class="fab fa-facebook-square fa-2x"></i></a></li>
+          <li><a href="https://www.linkedin.com/company/fusionauth/" target="_blank"><i class="fab fa-linkedin fa-2x"></i></a></li>
+          <li><a href="https://fusionauth.io/blog/feed.xml" target="_blank"><i class="fas fa-rss-square fa-2x"></i></a></li>
```
Comment (robotdan) [inline site/_includes/_footer.liquid:46]: Did you mean to change the icon size? `fa-3x` --> `fa-2x`?

Comment (mooreds) [reply site/_includes/_footer.liquid:46]: I did. Otherwise the 5th icon wraps the row onto two lines.

Comment (robotdan) [reply site/_includes/_footer.liquid:46]: Hmm. Ok. This changes the style for those icons, perhaps we just need to make some additional space in that row. 

@voidmain is doing some site / account work at the moment - any opinion if we make these smaller or widen the spot for these social icons.

Comment (voidmain) [reply site/_includes/_footer.liquid:46]: I'm going to manage the layout in style and remove the `fa-2x`. I'll merge this down and then fix it locally on master.

**Pull request merged**